man/systemd.netdev.xml

```diff
@@ -911,6 +916,23 @@
           <para>Accepts the same key as <literal>[FooOverUDP]</literal></para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Sequence=</varname></term>
+        <listitem>
+          <para> The <varname>Sequence</varname> Serializes the packets. Only applies for ERSPAN tunnel.
+          Defaults to <literal>unset</literal>.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>ErspanIndex=</varname></term>
+        <listitem>
+          <para>Specifies the ERSPAN index field. A 20 bit index/port number associated with the ERSPAN
+          traffic's source port and direction. This field is mandatory.
```

Comment:
maybe be even more explicit, and clarify what the valid numeric range is, i.e. the lowest and highest permissible integer, i.e. `1…1048575` iiuc?